docs/topics/secured-mission-intro-paragraph.adoc

```diff
@@ -3,6 +3,6 @@ Mission proficiency level:
 ifdef::docs-topic[xref:proficiency_advanced[*{proficiency-advanced}*].]
 ifndef::docs-topic[link:https://launcher.fabric8.io/docs/thorntail-runtime.html#proficiency_levels[*{proficiency-advanced}*^].]
 
-The {name-mission-secured} booster expands on the {name-mission-http-api} booster by securing a REST endpoint using link:https://access.redhat.com/products/red-hat-single-sign-on[{RHSSO}^]. {RHSSO} implements the link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.1/html/securing_applications_and_services_guide/openid_connect_3[Open ID Connect] protocol that is an extension of the OAuth 2.0 specification and uses it to issue access tokens to provide clients with various access rights to secured resources. Securing an application with SSO enables you to add security to your applications while centralizing the security configuration.
+The {name-example-secured} booster expands on the {name-example-http-api} booster by securing a REST endpoint using link:https://access.redhat.com/products/red-hat-single-sign-on[{RHSSO}^]. {RHSSO} implements the link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.1/html/securing_applications_and_services_guide/openid_connect_3[Open ID Connect] protocol that is an extension of the OAuth 2.0 specification and uses it to issue access tokens to provide clients with various access rights to secured resources. Securing an application with SSO enables you to add security to your applications while centralizing the security configuration.
```

Comment:
I can see the word "booster" here. @huntley, you might have changed it through another commit. Just pointing out in case it is not.